homeassistant/components/moehlenhoff_alpha2/__init__.py

```diff
@@ -0,0 +1,108 @@
+"""Support for the Moehlenhoff Alpha2."""
+import asyncio
+from datetime import timedelta
+import logging
+
+import aiohttp
+from moehlenhoff_alpha2 import Alpha2Base
+
+from homeassistant import exceptions
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.dispatcher import async_dispatcher_send
+from homeassistant.helpers.event import async_track_time_interval
+
+from .const import DOMAIN, SIGNAL_HEATAREA_DATA_UPDATED
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS = ["climate"]
+
+UPDATE_INTERVAL = timedelta(seconds=60)
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
+    """Set up a config entry."""
+    base = Alpha2Base(entry.data["host"])
+    try:
+        await base.update_data()
```

Comment:
We can use `coordinator.async_config_entry_first_refresh` instead. It will raise ConfigEntryNotReady by default if the first update fails.